packit_service/worker/helpers/testing_farm.py

```diff
@@ -320,16 +408,28 @@ def _payload_install_test(self, build_id: int, target: str, compose: str) -> dic
             },
         }
 
-    @staticmethod
-    def _payload_without_token(payload: Dict) -> Dict:
-        """Return a copy of the payload with token/api_key removed."""
-        payload_ = payload.copy()
-        payload_.pop("api_key")
-        payload_["notification"]["webhook"].pop("token")
-        return payload_
+    def check_comment_pr_argument_and_report(self) -> bool:
+        """
+        Check whether there are successful recent Copr builds for the additional PR given
+        in the test comment command argument.
+        """
+        if not self.copr_builds_from_other_pr:
+            self.report_status_to_tests(
+                description="We were not able to get any Copr builds for given additional PR. "
+                "Please, make sure the comment command is in correct format "
+                "`/packit test repo/namespace#pr_id`",
+                state=BaseCommitStatus.error,
+                url="",
+            )
+            return False
 
-    def is_fmf_configured(self) -> bool:
+        return True
```

Comment:
```suggestion
        if self.copr_builds_from_other_pr:
            return True

        self.report_status_to_tests(
            description="We were not able to get any Copr builds for given additional PR. "
            "Please, make sure the comment command is in correct format "
            "`/packit test repo/namespace#pr_id`",
            state=BaseCommitStatus.error,
            url="",
        )
        return False
```

↑ This diff looks very weird :eyes: